Tools/autotest/arduplane.py

```diff
@@ -976,17 +976,16 @@ def test_throttle_failsafe_fence(self):
 
         self.load_fence("CMAC-fence.txt")
 
-        self.set_parameter("FENCE_CHANNEL", 7)
-        self.set_parameter("FENCE_ACTION", 4)
-
-        self.wait_sensor_state(mavutil.mavlink.MAV_SYS_STATUS_GEOFENCE,
-                               present=True,
-                               enabled=False,
-                               healthy=True)
+        self.set_parameter("RC7_OPTION", 11) # AC_Fence uses Aux switch functionality
+        self.set_parameter("FENCE_ACTION", 4) # Fence action Brake
         self.set_rc_from_map({
             3: 1000,
             7: 2000,
-        })
+        }) # Turn fence on with aux function
+
+
+        self.mavproxy.expect("fence enabled")
```

Comment:
better wait for MSG_FENCE_STATUS, it is send when fence is enable !

Reply:
Ack - updated